packages/react/src/components/Authenticator/ConfirmSignIn/ConfirmSignIn.tsx

```diff
@@ -17,15 +21,16 @@ export const ConfirmSignIn = (): JSX.Element => {
   } = useAmplify(amplifyNamespace);
 
   const [state, send] = useAuth();
-  const isPending = state.matches('confirmSignIn.pending');
+  const actorState = getActorState(state);
```

Comment:
It looks like we aren't using the `state` from the `useAuth` now. Maybe we should change that variable to something like an underscore to signify that we intentionally aren't using it.

Question - does `useAuth` still provide a valid state for us to work with? And if it does, is the `getActorState` helper necessary?

Reply:
We don't need much from `useAuth` anymore, as we're mostly interested in child substates (like `signIn.confirmSignIn`), which are in actor. `useAuth` is now primarily used to get the nested context, which are at `state.context.actorRef.getSnapshot().context` which my helper function simplifies a bit.

I like underscore though! Let me do that cross-platform.

Reply:
- React: https://github.com/aws-amplify/amplify-ui/pull/180/commits/1960cab3bebefa929318d419e0eecf5764bda544
- Vue: https://github.com/aws-amplify/amplify-ui/pull/180/commits/dfeba88777421970baeeef7809b165969855b4a9


I think Angular is good because I set those context at the same place.